interface/super/rules/www/css/rules.css

```diff
@@ -157,3 +153,7 @@ root {
 input[type='text'].short {
   width: 80px;
 }
+
+.intervals span select {
+  width: inherit;
```

Comment:
The select field was using the paragraph's width instead of the span one so this uses whatever span width is computed by the browser.